rust/src/builtins/scriptlet_intercept/useradd.rs

```diff
@@ -35,6 +28,9 @@ pub(crate) fn entrypoint(args: &[&str]) -> Result<()> {
         .map(|s| s.parse::<u32>())
         .transpose()?;
     let group = matches.value_of("group");
+    let groups = matches
```

Comment:
I think it would be helpful to call this `supplementary_groups` (or something similar but shorter) and rename the existing one above as `primary_group` (or similar).

Reply:
Done!